pages/report-hazard/index.html

```diff
@@ -105,7 +105,8 @@
           </li>
         </ul>
       </nav>
-      <button id="backButton" class="back-button-mobile">&lt; Back</button>
+      <button id="backButton" class="back-button-mobile">Back</button>
+      <div class="underline"></div>
```

Comment:
can we rename this `underline` to `divider`?

Reply:
done